sycl/source/detail/event_impl.hpp

```diff
@@ -327,6 +335,8 @@ class event_impl {
   /// Store the command graph associated with this event, if any.
   /// This event is also be stored in the graph so a weak_ptr is used.
   std::weak_ptr<ext::oneapi::experimental::detail::graph_impl> MGraph;
+  /// Indicates that the event results from a command graph submission
+  bool EventFromSubmitedExecCommandBuffer = false;
```

Comment:
```suggestion
  bool MEventFromSubmitedExecCommandBuffer = false;
```
To align with other naming conventions for members.